### PR TITLE
Added link to table embed from files list

### PR DESCRIPTION
**Pull request by Jholder112233**
screen shot 2015-08-06 at 12 37 32

### CODE REVIEW

**Comment by Jholder112233** [comment]
@mattosborn happy for me to merge this?

File: src/templates/index.html [hunk 1]
```diff
@@ -36,6 +36,9 @@
 					<td><span class="permission permission--{{ f.domainPermissions }}">{{ f.domainPermissions }}</span></td>
 					<td>
 						<a class="docs" href="{{ f.urlDocs }}">docs</a> /
+						{% if 'tableDataSheet' in f.rawBody.sheets %}
+							<a class="table-embed" target="_blank" href="http://interactive.guim.co.uk/2015/07/table-testing/embed/index.html?spreadsheet={{ f.id }}">table url</a> /
```
Comment (mattosborn) [inline src/templates/index.html]: looks like a test url with a hard-coded date??

Comment (Jholder112233) [inline src/templates/index.html]: Good shout. Deployed this somewhere more permanent.

File: src/templates/index.html [hunk 2]
```diff
@@ -36,6 +36,9 @@
 					<td><span class="permission permission--{{ f.domainPermissions }}">{{ f.domainPermissions }}</span></td>
 					<td>
 						<a class="docs" href="{{ f.urlDocs }}">docs</a> /
+						{% if 'tableDataSheet' in f.rawBody.sheets %}
+							<a class="table-embed" target="_blank" href="http://interactive.guim.co.uk/tools/tables/embed/index.html?spreadsheet={{ f.id }}">table url</a> /
```
Comment (mattosborn) [inline src/templates/index.html]: https this link!

**Comment by mattosborn** [comment]
:+1: aside from https

**Pull request merged**